### PR TITLE
fixed bug 883

### PR DESCRIPTION
**Pull request by sofiiamatsyhin**

### CODE REVIEW

File: src/app/app.component.ts [hunk 1]
```diff
@@ -20,14 +20,19 @@ export class AppComponent implements OnInit  {
...
+    };
+  }
 
   ngOnInit() {
     this.languageService.setDefaultLanguage();
     this.navigateToStartingPositionOnPage();
     this.titleAndMetaTagsService.useTitleMetasData();
     this.searchSearch.searchSubject.subscribe(this.openSearchSubscription.bind(this));
     this.uiActionsService.stopScrollingSubject.subscribe(data => this.toggle = data);
+    this.router.events.subscribe((evt) => { if (evt instanceof NavigationEnd) { this.router.navigated = false; window.scrollTo(0, 0); } });
```
Comment (mplesha) [inline src/app/app.component.ts:35]: rewrite it according to the code style

File: src/app/app.component.ts [hunk 2]
```diff
@@ -20,14 +20,19 @@ export class AppComponent implements OnInit  {
...
   }
 
   private openSearchSubscription(isSearchExpanded: boolean): void {
```
Comment (mplesha) [inline src/app/app.component.ts:38]: There is an assignment statement in a ternary operator - it should be rewrite

**Comment by mplesha** [comment]
There is still an assignment statement in a ternary operator (the line 44)  - it should be rewrite

File: src/app/app.component.ts [hunk 3]
```diff
@@ -20,14 +20,24 @@ export class AppComponent implements OnInit  {
...
+    };
+  }
 
   ngOnInit() {
     this.languageService.setDefaultLanguage();
     this.navigateToStartingPositionOnPage();
     this.titleAndMetaTagsService.useTitleMetasData();
     this.searchSearch.searchSubject.subscribe(this.openSearchSubscription.bind(this));
     this.uiActionsService.stopScrollingSubject.subscribe(data => this.toggle = data);
+    this.router.events.subscribe((evt) => {
```
Comment (KimAT25) [inline src/app/app.component.ts:35]: you add the almost the same method in ngOnInit as navigateToStartingPositionOnPage(), why?
As you can use existed method and just update it,
image

**Pull request closed without merging**